test/test-fs-copyfile.c

```diff
@@ -199,8 +199,10 @@ TEST_IMPL(fs_copyfile) {
   touch_file(dst, 0);
   chmod(dst, S_IRUSR|S_IRGRP|S_IROTH); /* Sets file mode to 444 (read-only). */
   r = uv_fs_copyfile(NULL, &req, fixture, dst, 0, NULL);
+# ifndef __PASE__
   ASSERT(req.result == UV_EACCES);
   ASSERT(r == UV_EACCES);
+# endif
```

Comment:
So what does it return on PASE?

Reply:
> So what does it return on PASE?

It returns no error if running with qsecofr users (root-like).
```
/* On IBMi PASE, qsecofr users can overwrite read-only files */
```